### PR TITLE
hardware milestone 

### PR DESCRIPTION
**Pull request by Teodora1807**
### Pull Request Overview

This pull request is the hardware milestone (Kicad and photos of my work)

### Author

Signed-off-by: Teodora Dumitru <teodumitru18@gmail.com>

### CODE REVIEW

File: docs/project/2024/teodora.dumitru1807/BLOCK DIAGRAM.png
Comment (alexandruradovici) [inline docs/project/2024/teodora.dumitru1807/BLOCK DIAGRAM.png:1]: Please do not use spaces in the filenames.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Please fill the pull request template.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please do not use spaces in filenames.

**Review by alexandruradovici — APPROVED** [review]

**Review by alexandruradovici — DISMISSED** [review]

**Review by alexandruradovici — DISMISSED** [review]

**Review by TeodoraMiu — APPROVED** [review]

**Comment by Teodora1807** [comment]
I resolved the conflicts. Is everything ok?

**Review by alexandruradovici — APPROVED** [review]

**Review by TeodoraMiu — APPROVED** [review]

**Pull request merged**